.dotnet/src/Custom/Files/FileClient.Protocol.cs

```diff
@@ -209,4 +209,16 @@ public virtual ClientResult DownloadFile(string fileId, RequestOptions options)
         using PipelineMessage message = CreateDownloadFileRequest(fileId, options);
         return ClientResult.FromResponse(_pipeline.ProcessMessage(message, options));
     }
+
+    public virtual async Task<IncrementalUploadOperation> CreateIncrementalUploadAsync(BinaryContent content, RequestOptions options)
+    {
+        ClientResult result = await _internalUploadClient.CreateUploadJobAsync(content, options).ConfigureAwait(false);
+        return new IncrementalUploadOperation(_internalUploadClient, result.GetRawResponse());
+    }
+
+    public virtual IncrementalUploadOperation CreateIncrementalUpload(BinaryContent content, RequestOptions options)
+    {
+        ClientResult result = _internalUploadClient.CreateUploadJob(content, options);
+        return new IncrementalUploadOperation(_internalUploadClient, result.GetRawResponse());
+    }
```

Comment:
Consider adding a GetIncrementalUpload method that takes a `ContinuationToken`, following the pattern shown here:
- [assistantsClient.GetAssistantsAsync(pageToken) method](https://github.com/openai/openai-dotnet/pull/105/files#diff-8a5eb97415b5da3e5a1f4d701a0327137016101cb2342bc54c90923881e5c5d8R136-R151)
- [AssistantsPageToken](https://github.com/openai/openai-dotnet/pull/105/files#diff-ad84e4487a726fe8c3cc68aff06b4085fea792fa3d9090650f3407fa598f8aff)

This would take whatever state was needed to rehydrate the state of the operation -- if we're not storing an upload ID for the protocol method, then it likely doesn't even need to take a token to get an instance of it?  For convenience when we store the upload id, we would need that to be backed by a token with that and any other values that represented state to be rehydrated.

Reply:
Per offline conversation, we decided continuation would live in a static `Rehydrate` method on the subclient.